test/features/insertIf.test.tsx

```diff
@@ -139,7 +139,10 @@ describe('insertIf', () => {
 
   it('Recieves data from model', async () => {
```

Comment:
I just noticed: all these `Receives` are misspelled. May as well fix when you're adjusting these tests